### PR TITLE
Updated readme and example

### PR DESCRIPTION
**Pull request by lmroz**
-Updated readme (inc. snaptel).
-Renamed `mock-file` to `file`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,18 +37,13 @@ Build the plugin by running make in repo:
 ```
 $ make
 ```
-This builds the plugin in `/build/rootfs`
+This builds the plugin in `./build/`
 
 ### Configuration and Usage
-* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started).
```
Comment (katarzyna-z) [inline README.md]: Why did you remove this information? I think that it could be useful for users.

Comment (lmroz) [inline README.md]: I moved this to Example section.

Comment (katarzyna-z) [inline README.md]: ok

File: README.md [hunk 2]
```diff
@@ -37,18 +37,13 @@ Build the plugin by running make in repo:
...
-* Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-swap/blob/master/README.md#examples).
 
-## Documentation
-
-### Collected Metrics
 The path to the procfs can be provided in configuration as `proc_path`. If configuration is not provided, the plugin will use the default of `/proc`.
 
-It can be set in the snap global config that is loaded with snapd, e.g.:
+It can be set in the snap global config that is loaded with snapteld, e.g.:
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md [hunk 3]
```diff
@@ -66,37 +61,33 @@ It can be set in the snap global config that is loaded with snapd, e.g.:
...
+
+### Collected Metrics
+
 
 The list of collected metrics is described in [METRICS.md](https://github.com/intelsdi-x/snap-plugin-collector-swap/blob/master/METRICS.md).
 
 ### Examples
 
-Example running snap-plugin-collector-swap plugin and writing data to a file.
+Example of running snap swap collector and writing data to file.
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md [hunk 4]
```diff
@@ -66,37 +61,33 @@ It can be set in the snap global config that is loaded with snapd, e.g.:
...
 
-Make sure that your `$SNAP_PATH` is set, if not:
-```
-$ export SNAP_PATH=<snapDirectoryPath>/build
-```
-
-Other paths to files should be set according to your configuration, using a file you should indicate where it is located.
+Ensure [snap daemon is running](https://github.com/intelsdi-x/snap#running-snap):
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md [hunk 5]
```diff
@@ -66,37 +61,33 @@ It can be set in the snap global config that is loaded with snapd, e.g.:
...
+* initd: `sudo service snap-telemetry start`
+* systemd: `sudo systemctl start snap-telemetry`
+* command line: `sudo snapteld -l 1 -t 0 &`
 
-In one terminal window, open the snap daemon (in this case with logging set to 1,  trust disabled):
+Download and load snap plugins:
```
Comment (kindermoumoute) [inline README.md]: Snap*

File: README.md [hunk 6]
```diff
@@ -66,37 +61,33 @@ It can be set in the snap global config that is loaded with snapd, e.g.:
...
 ```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-swap/latest/linux/x86_64/snap-plugin-collector-swap
```
Comment (kindermoumoute) [inline README.md]: Precise this is the link for linux.

Comment (lmroz) [inline README.md]: This plugin has linux as requirement.

Comment (kindermoumoute) [inline README.md]: I missed that, my bad. :-)

**Comment by lmroz** [comment]
Is it GTG now?
